frontend/src/routes/Routes.tsx

```diff
@@ -24,6 +30,14 @@ export const AppRoutes = () => {
       <Route path={routes.WELCOME} element={<WelcomePage />} />
       <Route path={routes.SEARCH_PPC} element={<> TODO </>} />
       <Route path="*" element={<NotFound />} />
+      
+      {/* only for test error pages, need to be deleted */}
```

Comment:
Can you provide some context for this? Is there a related jira ticket?

Reply:
Jira ticket: https://jira.th.gov.bc.ca/browse/ORV2-967
I think these routes need to be removed after the code review, it's added just for test purpose.
Test with the following links:
https://onroutebc-586-frontend.apps.silver.devops.gov.bc.ca/notfound
https://onroutebc-586-frontend.apps.silver.devops.gov.bc.ca/loading
https://onroutebc-586-frontend.apps.silver.devops.gov.bc.ca/unauthorized
https://onroutebc-586-frontend.apps.silver.devops.gov.bc.ca/universalunauthorized
https://onroutebc-586-frontend.apps.silver.devops.gov.bc.ca/unexpected

Reply:
Ah okay, this is just to help with the code review. I understood this as adding routes to help the testing module.